benchmarks/Microsoft.AspNetCore.Razor.Performance/ProjectSystem/ProjectSnapshotManagerBenchmarkBase.cs

```diff
@@ -149,6 +141,14 @@ public override IProjectEngineFactory FindSerializableFactory(ProjectSnapshot pr
             {
                 throw new NotImplementedException();
             }
+
+            public override RazorProjectEngine Create(RazorConfiguration configuration, RazorProjectFileSystem fileSystem, Action<RazorProjectEngineBuilder> configure)
```

Comment:
This had to change because we need to be able to create the project engine from inside `ProjectState`.